SnapMD.ConnectedCare.ApiModels/PatientAccountInfo.cs

```diff
@@ -25,5 +25,8 @@ public class PatientAccountInfo
 
         public bool IsAuthorized { get; set; }
         public bool IsActive { get; set; }
+
+        public bool? IsPending { get; set; }
```

Comment:
@Number1MSFanboy @simplecodeOlga Is there a condition where a user can be both active and pending? What do you think about using a "Status" field instead, with Active, Inactive, Pending?